src/components/Highlight.vue

```diff
@@ -1,5 +1,4 @@
-<script>
-import { HIGHLIGHT_PRE_TAG, HIGHLIGHT_POST_TAG } from '../store';
+<script>import { HIGHLIGHT_PRE_TAG, HIGHLIGHT_POST_TAG } from '../store';
```

Comment:
The html plugin was already there previously. Tomorrow I will try without it or with the vue one.